nservicebus/messaging/databus/index.md

```diff
@@ -79,6 +93,22 @@ Automatically removing these attachments can cause problems in many situations.
 * If the data bus feature is used when publishing an event to multiple subscribers, neither the publisher nor any specific subscribing endpoint can determine when all subscribers have successfully processed the message allowing the file to be cleaned up.
 * If message processing fails, it will be handled by the [recoverability feature](/nservicebus/recoverability/). This message can then be retried some period after that failure. The data bus files need to exist for that message to be re-processed correctly.
 
+## Alternatives
+
+WARNING: Alternatives marked with ⚠️ are community project and are not maintained or supported by Particular Software.
+
+- Use a different transport or a different transport tier
+- Message Compression: Use message body compression which works well on text-based payloads like XML and Json or any payload (text or binary) that contains repetitive data
+  - [Message mutator example demonstrating message body compression](/samples/messagemutators/)
+  - Community maintained [NServiceBus.Comression ⚠️](https://github.com/ramonsmits/NServiceBus.Compression) with configurable compression thresholds
```

Comment:
you could also add them to https://docs.particular.net/nservicebus/community/ and link to that section alternatively

Reply:
Now linking the community page instead